cw_bitcoin/pubspec.yaml

```diff
@@ -56,6 +56,10 @@ dev_dependencies:
   hive_generator: ^1.1.3
 
 dependency_overrides:
+  ledger_flutter:
+    git:
+      url: https://github.com/cake-tech/ledger-flutter.git
+      ref: cake-v3
```

Comment:
**Minor:**
why override, why not just replace the package in the dependencies, since we are the ones who added it anyway, it's not a child package or anything

Reply:
Because the other Ledger packages depend on ledger_flutter from pub dev.

Reply:
let's check the issues Matthew mentioned in a separate PR next week please, add it to your note or on Jira